src/views/Swap/Swap.vue

```diff
@@ -792,6 +799,18 @@ export default {
         for (const [speed, fee] of Object.entries(assetFees)) {
           fees[chain][speed] = fees[chain][speed].plus(totalFees[fee.fee])
         }
+
+        if (asset === 'UST' && chain === 'LUNA') {
```

Comment:
Shouldn't be handled differently to bitcoin... should be implemented in `estimateFees` in `LiqualitySwapProvider`
One issue is that you don't have balance there, 
you can either pass it or I think better is if `getTaxFees` is on `TerraWalletProvider` and can check the balance internally. This way the parameters passed will be very similar to bitcoin